tests/system/feature_store/test_spark_engine.py

```diff
@@ -721,3 +729,49 @@ def test_error_is_properly_propagated(self):
                 spark_context=self.spark_service,
                 run_config=fs.RunConfig(local=False),
             )
+
+    @pytest.mark.parametrize("engine", ["spark"])
+    def test_get_offline_features_with_filter(self, engine):
```

Comment:
Not sure why you need to `parametrize` here, where the entire module is only dealing with Spark (it's called `test_spark_engine.py`...) and your only option for `engine` is `spark`. I assume this is just a copy-paste accident.